File: kombarika/com/dao/test/entity/Region.cs
namespace com.dao.test.entity;


using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;



[Table(name = "region")]
public class Region {

	[Column("nom_region")]
	string nomRegion { get; set; }
	[Key]
	[Column("id")]
	int id { get; set; }




	public Region(){}



}

File: kombarika/back/com/district/test/entity/Region.java
package com.district.test.entity;

import com.dao.annotation.*;
import com.dao.annotation.conf.ForeignType;


@Table(name = "region")
public class Region {
	@Column(name = "nom_region")
	String nomRegion;

	@PrimaryKey
	@GeneratedValue(autoIncrement = true)
	@Column(name = "id")
	Integer id;



	public Region(){}

	public String getNomRegion(){
		return this.nomRegion;
	}
	public void setNomRegion(String nomRegion){
		this.nomRegion = nomRegion;
	}

	public Integer getId(){
		return this.id;
	}
	public void setId(Integer id){
		this.id = id;
	}


}
